docs/known-issues.md

```diff
@@ -1 +1,25 @@
-Placeholder - see [the WIP known issues doc](https://brevi.link/unreal-gdk-known-issues).
\ No newline at end of file
+## Unreal GDK Known Issues
+
+Known issue = any major user-facing bug or lack of user-facing feature that:
+1. diverges from vanilla Unreal design or implementation (e.g. ordering of reliable RPCs), **OR**
+1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+
+| Issue | Date added | Ticket | Workaround? | Done? |
+|-------|-------------------|--------|-------------|-------|
+| Order of reliable RPCs is not respected in case of unresolved UObject* parameter. | 2018-06-14 | [UNR-363](https://improbableio.atlassian.net/browse/UNR-336) | Potential user-code workaround. | No|
+| Reliable multicast RPCs aren’t supported. | 2018-06-14 | | Ask yourself if you really need a reliable multicast RPC. | No |
```

Comment:
"As yourself if..." - this sounds like we're not planning to fix the issue, in which case I don't think it's fair to include it as a known issue.

Reply:
Also, I think it sounds a little bit rude - maybe rephrase, offering an alternative to using reliable multicast RPCs?

Reply:
Yes, it is more of an unsupported issue, we cover it in the trouble shooting guide, maybe we can remove it